tasks/build.js

```diff
@@ -55,9 +56,28 @@ async function api() {
     }
 }
 
+async function executeChildProcess(args) {
+    if (process.env.BUILD_CHILD) {
+        throw new Error('Infinite loop');
+    }
+    const env = {...process.env, BUILD_CHILD: '1'};
+    const child = fork(__filename, args, {env});
+    // Send SIGINTs as SIGKILLs, which are not ignored
+    process.on('SIGINT', () => {
+        child.kill('SIGKILL');
+        process.exit(130);
```

Comment:
We might need to clean it up, to not leave a partial-mess behind?

Reply:
I don't have a test for that, but I did try that and made sure that no processes were left alive. In fact, that used to be a problem, but I fixed it. the main process died and didn't kill the child.